de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/filesystem/IReferencePoint.java

```diff
@@ -0,0 +1,5 @@
+package de.fu_berlin.inf.dpp.filesystem;
+
+public interface IReferencePoint {
```

Comment:
How many times I have to repeat myself ? Write a f***** documentation for this interface. What is an ReferencePoint ? How should it be used ? Why it is only a marker interface ?

Reply:
I'm sorry for the missing doc...
I rewrote the issue https://github.com/saros-project/saros/issues/177. There is explained, why the IReferencePoint is only an empty interface and for what we need this.